code/game/objects/structures/table_frames.dm

```diff
@@ -21,59 +21,63 @@
 	var/framestack = /obj/item/stack/rods
 	var/framestackamount = 2
 
+/obj/structure/table_frame/Initialize(mapload)
+	. = ..()
+	register_context()
 
-/obj/structure/table_frame/wrench_act(mob/living/user, obj/item/I)
-	to_chat(user, span_notice("You start disassembling [src]..."))
```

Comment:
Since you've moved most of the de/construction tips over to balloon alerts away from to_chats, you'll want to make that consistent with reinforced tables as well, which are still using to_chats to show when they've been re/de-enforced.

Reply:
ah yeah this is about table _frames_, not the tables themselves- reinforced tables are entirely unrelated to this, and regular table `wrench_act(...)` also still uses the `to_chat(...)` calls.
but yeah prooooooobably good to have those be consistent while we're at it